index.ts

```diff
@@ -0,0 +1,15 @@
+function fizzbuzz() {
+    for (let i = 1; i < 101; i++) {
+        if (i % 15 === 0) {
+            console.log("FizzBuzz")
+        } else if (i % 3 === 0) {
```

Comment:
Similar comment on "maintainability" and "readability" as with Patryks :D 

If you take a peek at his PR you can see some of the considerations we might make when building systems to make sure they're more easily extensible https://github.com/Patryk-Kuchta/fizzbuzzC/pull/1#discussion_r1832356218

Reply:
Thanks, really interesting about the lowest common multiple thing. I've changed this to check for 3 and 5 separately, but haven't pulled that out into a separate function here, as it's something I've done in part 2.